File: abc/abc141/abc141_c.cs
// https://atcoder.jp/contests/abc141/submissions/7526690

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var input = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
        var N = input[0];
        var K = input[1];
        var Q = input[2];

        var list = Enumerable.Repeat(K, N).ToList();

        var A = new int[N];
        for (int i = 0; i < Q; i++)
        {
            A[int.Parse(Console.ReadLine()) - 1]++;
        }

        for (int i = 0; i < N; i++)
        {
            if (K - Q + A[i] > 0)
            {
                Console.WriteLine("Yes");
            }
            else
            {
                Console.WriteLine("No");
            }
        }
    }
}

File: abc/abc141/abc141_c.fs
// https://atcoder.jp/contests/abc141/submissions/12024273

let [| N; K; Q |] = stdin.ReadLine().Split() |> Array.map int

let A = Array.zeroCreate<int> N

[ 1..Q ]
|> List.map (fun _ -> stdin.ReadLine() |> int)
|> List.map (fun i -> A.[i - 1] <- A.[i - 1] + 1)
|> ignore

A
|> Array.map (fun x ->
    if K - Q + x > 0 then "Yes"
    else "No")
|> Array.iter stdout.WriteLine
